src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,172 @@
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { getPeople } from '../api';
+import { useEffect, useState } from 'react';
+import { Person } from '../types';
+import { useParams, useSearchParams } from 'react-router-dom';
+
+function preperedPeople(
```

Comment:
```suggestion
function getPreperedPeople(
```
it's a function - preparedPeople sounds like it's already an array of prepared people